In [3]:
from statsmodels.stats.power import FTestAnovaPower

# Assumptions for post-hoc power analysis

alpha = 0.05  # Common alpha level
effect_size = 0.25  # Small to medium effect size (typical in behavioral sciences)
n_groups = 3  # Three noise conditions (brown, pink, silence)
sample_size = 13  # Total number of participants

# Calculate the power
power_analysis = FTestAnovaPower()
power = power_analysis.solve_power(effect_size=effect_size, nobs=sample_size, alpha=alpha, k_groups=n_groups)

power*100

9.870560026624064

The post-hoc power analysis for your study, assuming a small to medium effect size (0.25), with 13 participants and three noise conditions (brown, pink, silence), at an alpha level of 0.05, yields a power of approximately 0.099.

This means there is only about a 9.9% chance of detecting an effect of this size, given your sample size and experimental design. In other words, the study is underpowered for detecting small to medium effects. Generally, a power of 0.80 (or 80%) is considered adequate, implying a 20% chance of a Type II error (failing to detect a true effect).

The low power suggests that the study might not have been able to reliably detect small or medium effects of noise conditions on cognitive performance. To increase the power in future studies, you would need to increase the sample size or adjust other aspects of the experimental design (like reducing variability or increasing the effect size).